.gitignore

```diff
@@ -9,3 +9,6 @@
 !/etc/build/.gitkeep
 
 /tests/Application/yarn.lock
+
+.idea/
```

Comment:
You shouldn't add these files to project .gitignore, only to your global .gitignore